packages/webcert/src/feature/certificate/Notifications/ResponsibleHospNameNotification.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react'
+import { useSelector } from 'react-redux'
+import userImage from '@frontend/common/src/images/user-image.svg'
+import { getResponsibleHospName, getIsUnsigned } from '../../../store/certificate/certificateSelectors'
+import { isCareAdministrator } from '../../../store/user/userSelectors'
+import styled from 'styled-components/macro'
+
+const Wrapper = styled.div`
+  margin-top: 16px;
+  margin-bottom: -20px;
+`
+
+const ResponsibleHospNameNotification: React.FC = () => {
+  const isUnsigned = useSelector(getIsUnsigned())
+  const isUserCareAdministrator = useSelector(isCareAdministrator)
+  const responsibleHospName = useSelector(getResponsibleHospName)
+
+  if (!isUnsigned || !isUserCareAdministrator || responsibleHospName === undefined) {
```

Comment:
If responsibleHospName is undefined, but the rest are true, shouldn't 'Ej angivet' be displayed?